packages/gasket-plugin-nextjs/lib/index.js

```diff
@@ -43,6 +43,17 @@ module.exports = {
         return { ...rest, serviceWorker, nextConfig };
       }
     },
+    async prompt(gasket, context, { prompt }) {
+      const { addSitemap } = await prompt([
```

Comment:
Add a safety check incase `addSitemap` was already added to the create context as we similarly bumped into with https://github.com/godaddy/gasket/pull/480
```suggestion
      if(!('addSitemap' in context)) {
        const { addSitemap } = await prompt([
```
Include a test for this condition as well.